themes/bootstrap5/templates/zotero/export.phtml

```diff
@@ -0,0 +1,10 @@
+<?php
+    // Set page title.
+    $this->headTitle($this->translate('Export'));
+
+    // Set up breadcrumbs:
+    $this->breadcrumbs()->set($this->translate('Export'));
+?>
+<h2><?=$this->transEsc('Export')?></h2>
```

Comment:
Should we use the (relatively new) page-title component here instead of hard-coding things? And even if hard-coding, shouldn't there be an h1 instead of an h2?